scss/includes/_shared_app_switcher.scss

```diff
@@ -0,0 +1,108 @@
+sl-dropdown.app-switcher-dropdown,
+sl-button.app-switcher-dropdown-trigger {
+  display: block;
+}
+
+sl-button.app-switcher-dropdown-trigger::part(base) {
```

Comment:
I would recommend taking advantage of SCSS nesting to organize these, which (IMO) DRYs it out and makes it much easier to read and understand. You can use `&` to chain nested styles onto the parent selector. Example:

```
sl-button.app-switcher-dropdown-trigger {
  &::part(base) {
    [part styles here]
  }
  &:hover::part(base) {
    [hover styles here]
  }
  &.dropdown-open::part(base) {
    [etc.]
  }
}